general/administration/installing.md

```diff
@@ -161,11 +168,12 @@ A community project to deploy Jellyfin on Kubernetes-based platforms exists [at
     --userns keep-id \
     --volume jellyfin-cache:/cache:Z \
     --volume jellyfin-config:/config:Z \
-    --volume jellyfin-media:/media:ro,z \
```

Comment:
I'm curious why remove this?

Reply:
Because it creates a volume which you don't want. In 99% of the cases you want to use a bind mount of your local media folder.

Reply:
IIRC that was intended to be an example of bind mounting a media volume, but I guess people copypasta'd it too much.